sample/buildSrc/build.gradle.kts

```diff
@@ -2,17 +2,18 @@
  * Copyright (c) 2020, Dropbox, Inc. All rights reserved.
  */
 plugins {
-    kotlin("jvm") version "1.4.10"
+    kotlin("jvm") version "1.6.10"
```

Comment:
Can we reuse the constants from `Dependencies`?

Reply:
Doesn't appear so. I think since that file is defined inside the `buildSrc` project we can't use it in the `build.gradle` for buildSrc